astropy/timeseries/periodograms/lombscargle/implementations/chi2_impl.py

```diff
@@ -13,10 +13,10 @@ def lombscargle_chi2(t, y, dy, frequency, normalization='standard',
 
     Parameters
     ----------
-    t, y, dy : array_like (NOT astropy.Quantities)
+    t, y, dy : array-like (NOT astropy.Quantities)
```

Comment:
Maybe move `NOT astropy.Quantites` to main text (and link properly?)

Reply:
✅